ipaserver/install/server/upgrade.py

```diff
@@ -2183,6 +2245,10 @@ def upgrade_configuration():
     setup_pkinit(krb)
     enable_server_snippet()
 
+    # Must be executed after certificate_renewal_update
+    # (see function docstring for details)
+    http_certificate_ensure_ipa_ca_dnsname(http)
```

Comment:
The return value is ignored.

Reply:
This was a vestige of an earlier attempt to get around the certmonger 7-day delay thing.  e.g. ipa-server-install could restart certmonger if the request was resubmitted.  I decided not to go down that path because it could not be made reliable and in fact, in my estimation represents a bigger risk of something very bad happening.  But I left the return True/False in there.

I removed it in the next update.